packages/component/src/BasicWebChat.js

```diff
@@ -113,6 +114,19 @@ function createAttachmentRenderer(additionalMiddleware) {
   };
 }
 
+// TODO: [P2] #2859 We should move these into <Composer>
```

Comment:
Please remember to file issues when you add TODOs.

Reply:
Updated #2859.